tokens/properties/global.json

```diff
@@ -306,6 +306,14 @@
             },
             "box-shadow": {
                 "value": "box-shadow"
+            },
+            "color": {
+                "value": "color"
+            }
+        },
+        "timing function": {
+            "ease": {
+                "value": "ease"
```

Comment:
This is currently not showing in Storybook. We need a new mdx file (09-transition-04...).

Reply:
agh! true